README.md

```diff
@@ -19,7 +19,7 @@ For the installation instructions, see [How to Install](./docs/user/README.md#ho
 Inspect the new available alpha commands by calling the `--help` option:
 
 ```sh
-kyma@v3 alpha --help
+kyma alpha --help
 ```
```

Comment:
Just to make sure: shouldn't you also remove or rephrase the warning that mentions v3 (lines 7-9)?

Reply:
This PR is about renaming the usage of the kyma-cli binary. This is still v3 but the name of the binary is changed. So other comments and phases should not be changed